afk/afk.go.tmpl

```diff
@@ -11,16 +11,15 @@
 */}}
 
 {{/* CONFIGURATION VALUE STARTS HERE */}}
-{{ $removeAfkOnMessage := true }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
+{{ $removeAfkOnMessage := false }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
 {{/* CONFIGURATION VALUE ENDS HERE */}}
 
-{{/* Dont edit below if you don't know what you're doing ! */}}
 {{ $time := 0 }}{{ $afk := dbGet .User.ID "afk" }}{{ $prefix := index (reFindAllSubmatches `.*?: \x60(.*)\x60\z` (execAdmin "Prefix")) 0 1 }}
 {{ if $args := .Args }}
     {{ if eq (lower (index $args 0)) (print $prefix "afk") }}
         {{ if and (ge (len $args) 2) (not $afk) }}
             {{ $args = (slice .Args 1) }}
-            {{ $message := "" }}
+            {{ $message := "No reason setted" }}
```

Comment:
A more sensible afk reason when none is set:
```suggestion
            {{ $message := "afk" }}
```

Reply:
I don't think we should be using a default unless strictly necessary. See review comment later on.

Reply:
I think we shouldn't have a default message at all unless absolutely necessary, see my last review comment.